### PR TITLE
fix: Remove createLegacyApp, add render to index

### PR DESCRIPTION
**Pull request by taustad**

### CODE REVIEW

**Review by ahmedoabdi — COMMENTED** [review]
Please handle codacy identation warnings

File: pipelines/templates/stage-deploy-fusion.yml [hunk 1]
```diff
@@ -70,12 +70,6 @@ stages:
             cancelTimeoutInMinutes: 60
             timeoutInMinutes: 60
             steps:
-                - template: step-updateradixdynatraceConfig.yml
```
Comment (ahmedoabdi) [inline pipelines/templates/stage-deploy-fusion.yml:73]: Are we not using dynatrace logging in frontend?

**Review by ahmedoabdi — APPROVED** [review]
LGTM!

File: frontend/tsconfig.json [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
-  ]
-}
+    "typedocOptions": {
+        "entryPoints": [
+            "src/pages/Dashboard.tsx",
+            "src/pages/contract/GeneralPage.tsx"
+        ],
+        "out": "docs"
+    }
+}
```
Comment (DanielBohme) [inline frontend/tsconfig.json:32]: missing new line at end of file?

File: frontend/src/index.tsx [hunk 3]
```diff
@@ -1,24 +1,12 @@
...
-        buildUrl: (context: Context | null) => (context ? `/${context.id}` : ""),
-        getContextFromUrl: (url: string) => url.split("/")[1],
-    },
-    name: "DCD Concept App",
+registerLegacy("conceptapp", {
+    render: createComponent(App, configurator),
+    AppComponent: App,
 })
 
 if (module.hot) module.hot.accept()
```
Comment (DanielBohme) [inline frontend/src/index.tsx:12]: missing new line at end of file?

**Pull request merged**